src/features/user_profile/components/LoginForm/LoginForm.module.css

```diff
@@ -2,7 +2,7 @@
   z-index: var(--modal);
 }
 .modalCard {
-  box-shadow: none;
+  box-shadow: none !important;
```

Comment:
can we  use nesting instead? `!important` is the last thing we use

Reply:
research on the issue https://kontur.fibery.io/Tasks/User_Story/Disaster-Ninja-Frontend-Architecture-Improvements-v1-1523/FE-architecture-issues-2912/anchor=CSS-order-differs-on-dev-prod-(builder-i--7773343b-bfaf-42c5-b6bf-f7f58d26bcb2